src/index.ts

```diff
@@ -64,9 +66,34 @@ debug("Launching Impftermin");
         await page.addScriptTag({
           content: `new Audio("data:audio/wav;base64,${SOUND_BASE64}").play();`,
         });
-        // stop scraper for 25 minutes after a hit
-        setTimeout(() => runChecks(), 1000 * 60 * 25);
-        return;
+		// Try Book dates automatically
+		if (await bookAppointment(
+		  page, 
+		  entry.title!, 
+		  entry.firstname!, 
+		  entry.lastname!, 
+		  entry.zip!, 
+		  entry.city!, 
+		  entry.street!, 
+		  entry.streetnumber!,
+		  entry.mobile!,
+		  entry.email!,
+		  entry.earliestdate!,
+		  entry.latestdate!
```

Comment:
See above - it may be enough to pass in the `entry` object instead of extracting each variable individually?

Reply:
Agreed!